R/lifecycle_admiral.R

```diff
@@ -30,14 +29,22 @@
 #'
 #' @export
 deprecate_inform <- function(
+    when,
+    what,
+    with = NULL,
+    details = NULL,
+    id = NULL,
     env = caller_env(),
-    user_env = caller_env(2),
-    ...) {
+    user_env = caller_env(2)) {
```

Comment:
I mentioned this when we chatted, but will also write it here so we don't forget! Since we don't export `caller_env()`, we should prefix is with `rlang::caller_env()`

Reply:
Updated - thanks for the reminder!